File: UMA.App.IdentityManager/Role/Queries/UpdateRoleQuery.cs
﻿using MediatR;
using System;
using UMA.App.IdentityManager.Role.Models;

namespace UMA.App.IdentityManager.Role.Queries
{
    public class UpdateRoleQuery : IRequest<UpdateRoleResult>
    {
        public Guid Id { get; set; }
        public string Name { get; set; }
        public string Description { get; set; }
    }
}

File: UMA.Web.Angular/UMA-Web-Angular/src/app/_entities/role/query/updateRole.query.ts
import { UpdateRoleViewModel } from '../reponse/updateRole.viewmodel';
import { IRequest } from 'src/app/_models/interfaces/IRequest';

export class UpdateRoleQuery implements IRequest<UpdateRoleViewModel> {
  _response?: UpdateRoleViewModel;
  id: string;
  name: string;
  description: string;
}
